src/js/panels.js

```diff
@@ -0,0 +1,40 @@
+/*
+
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+(function ($, fluid) {
+
+    "use strict";
+
+    fluid.registerNamespace("gpii.firstDiscovery.panel");
+
+    fluid.defaults("gpii.firstDiscovery.panel.audio", {
+        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        preferenceMap: {
+            "gpii.firstDiscovery.audio": {
+                "model.audio": "default",
+                "range.min": "minimum",
+                "range.max": "maximum"
+            }
+        }
+    });
+
+    fluid.defaults("gpii.firstDiscovery.panel.contrast", {
+        gradeNames: ["fluid.prefs.panel", "autoInit"],
+        preferenceMap: {
+            "gpii.firstDiscovery.contrast": {
+                "model.value": "default",
+                "controlValues.theme": "enum"
+            }
+        }
+    });
+
+})(jQuery, fluid);
```

Comment:
The textSize panel hasn't been defined since the branch currently is using the default one from the infusion to match up the temporary template which is a copy paste from the infusion too. A new textSize panel will need to be created once the actual template is in place.